views/index.html

```diff
@@ -95,31 +95,29 @@ <h1 class="animated fadeInUp">CVR</h1>
 <code>
 #!/bin/bash
 
-# Required Travis config:
-# - Add the repo token in the variable CVR_TOKEN as a secure property in the Travis configuration
```

Comment:
I could go either way on this one. It says Travis "doesn't make the token available on pull requests," and indeed, the token isn't used in the sample script. I feel that the text previously explained the contents of the `$TRAVIS_PULL_REQUEST` conditional, and now explains why the token isn't mentioned anywhere in the script.